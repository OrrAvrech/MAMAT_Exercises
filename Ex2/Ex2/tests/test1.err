Command not found.
Insert execution failed.
Insert execution failed.
Add execution failed.
Add execution failed.
Add execution failed.
Add execution failed.
Add execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Depart execution failed.
Remove execution failed.
Remove execution failed.
Remove execution failed.
