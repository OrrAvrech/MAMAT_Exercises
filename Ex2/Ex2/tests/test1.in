
print
Print
Insert -1 I
Insert 1 P
Insert 1 D
Add 1 D JFK R
Add 1 D AAA R
Add 2 D AAA R
Add 3 D BBB R
Add 4 D CCC R
Insert 2 I
Add 50 I CCC E
Depart
Print
Add 50 I CCC E


Print
Insert 3 D
Add 60 D KFC E
Add 61 I KFC E
Add 62 D ABC E
Depart
Insert 4 D
Print
Add -1 D KAK R
Add 11 D aBC R
Add 12 D AbC R
Print
Add 13 D ABc R
Add 13 D ABC R
Print
Add 14 D ARK R
Add 15 D JFK R
Print
Add 16 D JFK R
Add 17 D JFK E
Print
Delay JFK
Print
Delay ABC
Print
Depart
Depart
Depart
Print
Depart
Depart
Remove 4
Print
Depart
Depart
Depart
Print
Depart
Depart
Depart
Print
Depart
Print
Depart
Depart
Depart
Depart
Print
Remove 4
Print
Remove 1
Remove 2
Remove 3
Remove 4
Remove 5
Insert 1 D
Add 1 D JFK R
Add 2 D ABC E
Print
Change JFK AAA
Print
Delay AAA
Print