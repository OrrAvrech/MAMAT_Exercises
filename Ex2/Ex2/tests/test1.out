Airport status:

Airport status:
Runway 1 domestic
4 flights are waiting:
Flight 1 D JFK R
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
0 flights are waiting:

Airport status:
Runway 1 domestic
4 flights are waiting:
Flight 1 D JFK R
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
1 flights are waiting:
Flight 50 I CCC E

Airport status:
Runway 1 domestic
3 flights are waiting:
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
2 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Runway 4 domestic
0 flights are waiting:

Airport status:
Runway 1 domestic
3 flights are waiting:
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
2 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Runway 4 domestic
0 flights are waiting:

Airport status:
Runway 1 domestic
3 flights are waiting:
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
2 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Runway 4 domestic
1 flights are waiting:
Flight 13 D ABC R

Airport status:
Runway 1 domestic
3 flights are waiting:
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
3 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Flight 15 D JFK R
Runway 4 domestic
2 flights are waiting:
Flight 13 D ABC R
Flight 14 D ARK R

Airport status:
Runway 1 domestic
4 flights are waiting:
Flight 17 D JFK E
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
3 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Flight 15 D JFK R
Runway 4 domestic
3 flights are waiting:
Flight 13 D ABC R
Flight 14 D ARK R
Flight 16 D JFK R

Airport status:
Runway 1 domestic
4 flights are waiting:
Flight 17 D JFK E
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
3 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Flight 15 D JFK R
Runway 4 domestic
3 flights are waiting:
Flight 13 D ABC R
Flight 14 D ARK R
Flight 16 D JFK R

Airport status:
Runway 1 domestic
4 flights are waiting:
Flight 17 D JFK E
Flight 2 D AAA R
Flight 3 D BBB R
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
3 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Flight 15 D JFK R
Runway 4 domestic
3 flights are waiting:
Flight 14 D ARK R
Flight 16 D JFK R
Flight 13 D ABC R

Airport status:
Runway 1 domestic
1 flights are waiting:
Flight 4 D CCC R
Runway 2 international
2 flights are waiting:
Flight 50 I CCC E
Flight 61 I KFC E
Runway 3 domestic
3 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Flight 15 D JFK R
Runway 4 domestic
3 flights are waiting:
Flight 14 D ARK R
Flight 16 D JFK R
Flight 13 D ABC R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 international
1 flights are waiting:
Flight 61 I KFC E
Runway 3 domestic
3 flights are waiting:
Flight 60 D KFC E
Flight 62 D ABC E
Flight 15 D JFK R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 international
0 flights are waiting:
Runway 3 domestic
1 flights are waiting:
Flight 15 D JFK R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 international
0 flights are waiting:
Runway 3 domestic
0 flights are waiting:

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 international
0 flights are waiting:
Runway 3 domestic
0 flights are waiting:

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 international
0 flights are waiting:
Runway 3 domestic
0 flights are waiting:

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 international
0 flights are waiting:
Runway 3 domestic
0 flights are waiting:

Airport status:
Runway 1 domestic
2 flights are waiting:
Flight 2 D ABC E
Flight 1 D JFK R

Airport status:
Runway 1 domestic
2 flights are waiting:
Flight 2 D ABC E
Flight 1 D AAA R

Airport status:
Runway 1 domestic
2 flights are waiting:
Flight 2 D ABC E
Flight 1 D AAA R

